### ISSUE TITLE
Constantly getting the "Failed to get list of R functions" error

### ISSUE THREAD

**Issue by rnd195** [REditorSupport/vscode-R#877]
**Describe the bug**
I am constantly getting the following two errors (also see screenshot-1):

```
Failed to get list of R functions. Make sure that `jsonlite` is installed and r.rpath.windows points to a valid R executable.
```

```
Command failed: "C:\Program Files\R\R-4.1.2\bin\R.exe" --silent --no-save --no-restore --slave -f "c:\Users\MyUserName\.vscode-oss\extensions\ikuyadeu.r-2.3.3\R\help\getAliases.R" > "C:\Users\MyUserName\AppData\Local\Temp\vscode-R-aliasesgNaXbS\aliases.json"
```

Here's the thing. The package `jsonlite` is installed (see screenshot-2) and r.rpath.windows does point to a valid R executable. 

I found a few similar issues, mainly #865 (updating to a newer version v VSCode did not work for me), and #829, which I am not sure whether the original poster's solution applies to my case, since I am not running VSCode on a server.

What I tried to fix the issue:
- updating to VSCode Insiders 1.63 version (did not work)
- adding R.exe to PATH (did not help)
- completely removing the `languageserver` package (errors still popped-up)
- running getAliases.R in RStudio (this worked without a problem)
- copy-pasted the command (the one from the error window) into Win+R and ran the command (it opened up cmd and seemed to work)

**To Reproduce**

1. Open any R file (in my case, ipynb with R kernel)
2. Hover over any function
3. Wait for a couple of seconds
4. Errors start popping-up

## Do you want to fix by self? (We hope your help!)

No

## (If yes,) what kind of help do you want? (e.g. Which file should I fix, Survey (related documents)

I'd like to know which file causes this failure. Or, generally, I'll be happy for any leads to resolve this issue.

(If related)setting.json

```json
"r.rterm.windows": "C:\\Program Files\\R\\R-4.1.2\\bin\\R.exe",
"r.rpath.windows": "C:\\Program Files\\R\\R-4.1.2\\bin\\R.exe"
```

**Screenshots**
screenshot-1:
![screenshot-1](https://user-images.githubusercontent.com/49758429/143047798-3ce76a63-8280-4fa2-8ce8-001a960a39c2.png)
screenshot-2:
![screenshot-2](https://user-images.githubusercontent.com/49758429/143047811-8eccfc15-43d8-484d-a381-cdf83ee0d040.png)

**Environment (please complete the following information):**

- OS: Windows 10 64-bit
- VSCode Version: 1.62.3
- R Version: 4.1.2
- vscode-R version: 2.3.3

Thank you for any input.

**Comment by rnd195** [comment]
I think I've managed to solve the issue by unchecking the option "Help Panel: Enable Hover Links" in the extension's settings. I mean, the errors are no longer popping-up, so that's a win for me, but I am not sure how good of a solution this is. Anyway, closing!

**Comment by eitsupi** [comment]
@rnd195 Is the path separator correct?
I think you need to use a slash `/`, or if you use a backslash, escape it with a backslash (`\\`).
Like that
![image](https://user-images.githubusercontent.com/50911393/143146117-744c67fe-691e-4f71-b4c5-cc3cbe6fc003.png)
https://code.visualstudio.com/docs/getstarted/settings

**Comment by danni-storm** [comment]
This seems to have solved the issue for me as well. I had a power blip in the middle of running code and the same errors referencing temp files and requesting I install jsonlite started streaming across VSCode when I opened it up again. I tried a complete fresh install of R and VSCode but the problem persisted. Turning off the "Hover Links" feature seemed to have worked but I am hoping for a resolution that doesn't involve me turning off a useful feature 😄. 

Also, getting familiar with GitHub as I use it more so I apologize if posting on a closed thing is bad form.
